moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -39,7 +39,7 @@
 #include <geometry_msgs/msg/pose.hpp>
 #include <moveit_msgs/msg/move_it_error_codes.hpp>
 #include <moveit/macros/class_forward.h>
-#include "rclcpp/rclcpp.hpp"
+#include "node_interface/node_interface.h"
```

Comment:
You should rename this to moveit_node_interface.h to make clear, that this is not part of rclcpp